requirements/dev-requirements.in

```diff
@@ -12,6 +12,7 @@ wheel==0.29.0
 transifex-client==0.12.4
 modernize
 gnureadline==6.3.8
+git-branch-builder
```

Comment:
nit: git commands are usually verbs or verb phrases. Ex: `git-build-branch`. Is it too late to change that?

Reply:
:white_check_mark: Thanks!

Reply:
Is there still some way to go in the renaming? The pypi entry and the repo https://github.com/dimagi/git-build-branch are renamed, but looking at the code _in_ the repo it looks like the main directory as well as the executable are still called git-branch-builder.

Reply:
I forgot to push my changes. The repo is now updated.